Assets/Scripts/Control&Input/PlayerMovement.cs

```diff
@@ -186,18 +169,15 @@ private void UpdatePosition(Vector3 input)
                     // Strafe slightly with less drag.
                     body.drag = airDrag;
                     body.AddForce(input * strafeForceInAir, ForceMode.VelocityChange);
-                    body.AddForce(Vector3.down * MarsGravity, ForceMode.Acceleration);
+                    body.AddForce(Vector3.down * marsGravity * 3.5f, ForceMode.Acceleration);
                     if (!IsInAir()) state = PlayerState.GROUNDED;
                     break;
                 }
             case PlayerState.GROUNDED:
                 {
                     // Strafe normally with heavy drag.
                     body.drag = drag;
-                    // Walk along ground normal (adjusted if on heavy slope).
-                    var groundNormal = GroundNormal();
-                    var direction = Vector3.ProjectOnPlane(input, groundNormal);
-                    body.AddForce(direction * strafeForce, ForceMode.VelocityChange);
+                    body.AddForce(input * strafeForce, ForceMode.VelocityChange);
```

Comment:
Um, why did you remove the slope movement code? 🤔

Reply:
???, must've been some merge conflicts, will fix